packages/core/src/App/Containers/CloseUKAccountModal/close-uk-account-modal.scss

```diff
@@ -0,0 +1,101 @@
+.close-uk-account {
+    &--is-fullscreen {
+        max-width: 80vw;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        flex-direction: column;
+        margin: 0 auto;
+        .close-uk-account__content {
+            max-width: 800px;
```

Comment:
For width and height properties, it is recommended to use `Rem` to make sure it will handle different screen sizes.